11/umbraco-cms/reference/routing/request-pipeline/outbound-pipeline.md

```diff
@@ -307,6 +307,10 @@ namespace RoutingDocs.UrlProviders
 }
 ```
 
+{% hint style="info" %}
+If you want to have multiple UrlProvider, you can add them one after the other with multiple `Insert` methods. Umbraco will cycle though all the providers registered until it finds one that doesn't return `null`. If all custom url providers return `null` it will fall back to the default url provider. Note that the last added with `Insert` is the first being executed. 
```

Comment:
```suggestion
If you want to have multiple URL providers, you can add them one after the other with multiple `Insert` methods. Umbraco will cycle through all the providers registered until it finds one that doesn't return `null`. If all custom URL providers return `null` it will fall back to the default URL provider. The last added with `Insert` is the first that will be executed. 
```